decentraland/unity-renderer
# feat: Manage the unread messages notifications for channels

## What does this PR change?
Fix https://github.com/decentraland/unity-renderer/issues/2398

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/manage-the-unread-messages-notifications
2. Notice the unread notifications behaviour (in both taskbar and the specific channel rows)

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
